packages/commonwealth/client/scripts/views/pages/CommunityManagement/Contests/EmptyContestsList/EmptyCard/EmptyCard.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import { CWText } from 'views/components/component_kit/cw_text';
+import { CWButton } from 'views/components/component_kit/new_designs/CWButton';
+
+import './EmptyCard.scss';
+
+interface EmptyCardProps {
+  img: string;
+  title: string;
+  subtitle: string;
+  button: {
+    label: string;
+    handler: () => void;
+  };
+}
+
+const EmptyCard = ({ img, title, subtitle, button }: EmptyCardProps) => {
+  const handleOpenLink = () => {
+    window.open('https://commonwealth.im/', '_blank');
```

Comment:
unsure, but should this be dynamic or is this a mock?

Reply:
it's just a mock for now

Reply:
but it is not gonna be dynamic, at least for now